opam-publish.opam

```diff
@@ -11,16 +11,24 @@ homepage: "https://github.com/ocaml/opam-publish"
 bug-reports: "https://github.com/ocaml/opam-publish/issues"
 license: "LGPL-2.1 with OCaml linking exception"
 dev-repo: "git+https://github.com/ocaml/opam-publish.git#2.0"
-build: [ "jbuilder" "build" "-p" name ]
+build: [ "dune" "build" "-p" name "-j" jobs ]
 depends: [
+  "cmdliner" {build}
+  "dune" {build}
+  "lwt_ssl"
+  "ocaml" {>= "4.02.3"}
```

Comment:
Unusually, perhaps this should be `{build}` although really what's meant is `{!ocaml:native | build}` which I don't think is permitted.